qa/L1_FW_iterators_separate_executor/test.h

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -e
+# used pip packages
+pip_packages="torchvision mxnet-cu##CUDA_VERSION##"
+
+pushd ../..
+
+cd dali/test/python
+
+NUM_GPUS=$(nvidia-smi -L | wc -l)
+
+test_body() {
+    # We install it manually here because we want to limit to one version of TF
```

Comment:
change of plans :)
You can specify `tensorflow-gpu` in pip_packages and then do `one_config_only=true`
```
pip_packages="tensorflow-gpu torchvision mxnet-cu##CUDA_VERSION##"
one_config_only=true
```

Reply:
Done